.github/docker/Dockerfile.gorgone-testing-bullseye

```diff
@@ -0,0 +1,30 @@
+FROM debian:bullseye
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# fix locale
+RUN bash -e <<EOF
+
+apt-get update
+apt-get install -y locales libcurl4-openssl-dev curl wget zstd jq lsb-release mariadb-client
+rm -rf /var/lib/apt/lists/*
+localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+apt-get clean
+
+EOF
+
+ENV LANG en_US.utf8
+
+RUN bash -e <<EOF
+apt-get update
+# Install Robotframework
+apt-get install -y python3 python3-dev python3-pip ca-certificates apt-transport-https software-properties-common wget gnupg2
+pip3 install robotframework robotframework-examples
+
+echo "deb https://packages.centreon.com/apt-standard-23.10-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list
+echo "deb https://packages.centreon.com/apt-plugins-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon-plugins.list
+
+wget -O- https://apt-key.centreon.com | gpg --dearmor | tee /etc/apt/trusted.gpg.d/centreon.gpg > /dev/null 2>&1
+apt-get update
+
+EOF
```

Comment:
we don't really want -q because we want the error to be printed if we can't download the file.
using tee allow to know exactly where the key is added for later management (if we want to remove it for exemple)